CRM/Volunteer/Upgrader.php

```diff
@@ -116,7 +111,12 @@ public function installProjectRelationships() {
     );
 
     foreach ($options as $opt) {
-      civicrm_api3('OptionValue', 'create', $optionDefaults + $opt);
+      $getOptionValues = civicrm_api3('OptionValue', 'get', $optionDefaults + $opt);
```

Comment:
Meh. We probably do want those values to be set that way all the time, and it's highly unlikely the values would come from an API call since we are installing new options, but I agree that it's intuitive that defaults should be mutable, so I've pushed another commit.